eZ/Publish/Core/Persistence/Cache/UserHandler.php

```diff
@@ -89,14 +93,16 @@ public function loadByEmail($email)
         $this->logger->logCall(__METHOD__, array('email' => $email));
         $users = $this->persistenceHandler->userHandler()->loadByEmail($email);
 
-        $cacheItem->set($users);
-        $cacheTags = [];
-        foreach ($users as $user) {
-            $cacheTags[] = 'content-' . $user->id;
-            $cacheTags[] = 'user-' . $user->id;
+        if (!empty($users)) {
```

Comment:
isn't this approach mutually exclusive from the fix above?
This basically means we are no longer caching lookups for not found users.

Reply:
In second thought I've realized that in fact, it makes sense to cache these empty arrays. Therefore, I've removed this change.